### ISSUE TITLE
Make RPC tests that use '-minrelaytxfee=0` after #6542 use a more realistic fee instead

### ISSUE THREAD

**Issue by daira** [zcash/zcash#6550]
#6542 changes `-minrelaytxfee` to actually enforce the minimum relay fee, removing the concept of "free transactions".

In order to repair tests that broke as a result, I've added `-minrelaytxfee=0` to many of the RPC tests. Needing to do this indicates that these tests were trying to send transactions that would not now be relayed on mainnet and testnet, because they were paying an insufficient fee: less than $\mathsf{min}(1000, \mathsf{floor}(100 \times \mathit{txSerializedSize} / 1000))$ zatoshis.

The tests should be updated to use realistic fees &mdash; preferably the ZIP 317 conventional fee, in cases where they're not intended to be testing other fee choices.
